manifests/native.json

```diff
@@ -215,6 +215,14 @@
       "replacement": "Array.prototype.lastIndexOf",
       "mdnPath": "Global_Objects/Array/lastIndexOf",
       "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "bluebird",
+      "nodeVersion": "0.12.0",
+      "replacement": "Promise",
```

Comment:
`bluebird` also contains many helpers, so it would be better to create a page and explain it.

Reply:
fair point 🤔 i thought it belongs as a `native` replacement since we can define when `Promise` became available etc

but maybe i made a mistake here and it belongs in `preferred` after all?

what do you think?

Reply:
Since in this case is possible to create a page with an explanation, I think that `preferred` is a better place.

Reply:
@benmccann would you mind opening a new PR with your original change, and undo this one?

i didn't realise bluebird has various other exports

Reply:
done here: https://github.com/es-tooling/module-replacements/pull/51